B827EBFFFE20B616.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE20B616",
+    "servers": [
+      {
+        "server_address": "ttn-router-eu",
```

Comment:
That's not a valid server address, please check the list of valid ones here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses